corehq/apps/data_interfaces/templates/data_interfaces/explore_case_data.html

```diff
@@ -0,0 +1,52 @@
+{% extends "hqwebapp/two_column.html" %}
```

Comment:
How are you imagining this template works for the second report made in this framework? Would they also provide an entire custom template or were you planning on trying to use template inheritance (or composition) to enable more sharing at the template level?